pom.xml

```diff
@@ -845,20 +785,10 @@
                 <version>${atomikos-version}</version>
             </dependency>
             <dependency>
-                <groupId>org.wso2.orbit.org.quartz-scheduler</groupId>
-                <artifactId>quartz</artifactId>
-                <version>${quartz.version}</version>
-                <exclusions>
-                    <exclusion>
-                        <groupId>com.zaxxer</groupId>
-                        <artifactId>HikariCP</artifactId>
-                    </exclusion>
-                </exclusions>
-            </dependency>
-            <dependency>
-                <groupId>commons-net</groupId>
-                <artifactId>commons-net</artifactId>
-                <version>${commons-net.version}</version>
+                <groupId>org.apache.mina</groupId>
```

Comment:
I'm assuming this was earlier transitive dependency and you have added it explicitly?

Reply:
Yes